nix/modules/srcCleaned.nix

```diff
@@ -1,4 +1,4 @@
-{ self, lib, inputs, ... }:
+{ self, lib, inputs, config, ... }:
```

Comment:
Inconsistency here, should this use `@flake` here and access `flake.config.reconciledInputs`

Reply:
:+1: